internal/venice-common/src/main/java/com/linkedin/venice/pushmonitor/ExecutionStatusWithDetails.java

```diff
@@ -30,4 +40,14 @@ public String getDetails() {
   public boolean isNoDaVinciStatusReport() {
     return noDaVinciStatusReport;
   }
+
+  /**
+   * N.B.: This is not available in older controller versions, and not available for all statuses (e.g. if the status is
+   * {@link ExecutionStatus#NOT_CREATED}, it does not have this timestamp).
+   *
+   * @return the UNIX Epoch timestamp of the last update to the status, or null if not available.
```

Comment:
nit: return some sentinel value for non-existing status instead of null. might save us from NPE in future.

Reply:
We could, although mis-handling a sentinel value is still a bug, even though it does not cause a NPE. IMHO the main benefit of sentinel value instead of boxed/nullable types is to limit memory usage, but in this case I think that doesn't matter... It could also be an Optional, though that also does not prevent bugs in my opinion. BTW, do you know how does Jackson handle new fields getting added? Is there a way to set the default value when dealing with payloads encoded by older code (i.e. which are missing the new field)? If it is a boxed type, then I'm pretty sure a missing field is going to be null, but if it's a primitive type, IDK...